tests/pytorch/nightly/fs-transformer.libsonnet

```diff
@@ -203,10 +206,9 @@ local utils = import 'templates/utils.libsonnet';
   },
   configs: [
     transformer + v3_8 + functional_no_save + timeouts.Hours(1),
-    transformer + v3_8 + convergence + timeouts.Hours(25),
     transformer + v3_8 + convergence + timeouts.Hours(25) + tpuVm,
-    transformer + v3_8 + checkpoint_local + timeouts.Hours(2),
-    transformer + v3_8 + checkpoint_gcs + timeouts.Hours(2),
+    transformer + v2_8 + checkpoint_local + timeouts.Hours(2),
```

Comment:
hmm, if checkpoint test is running using transformer, I am afraid it will OOM unless we update the config

Reply:
Good point. Changing it back to v3-8